compiler-core/src/language_server/tests/snapshots/gleam_core__language_server__tests__action__rename_invalid_bit_array_pattern.snap

```diff
@@ -0,0 +1,7 @@
+---
+source: compiler-core/src/language_server/tests/action.rs
+expression: "apply_first_code_action(code, 1)"
```

Comment:
Please put the original src as the expression so we can tell what the snapshot is for. It is quite challenging to debug quickly if you can't see what the input is!